html/static/script.js

```diff
@@ -67,13 +67,52 @@ function saveNotes(element, uid) {
   })
 }
 
+function clearNotesMissing(element, dataset_name, campaign) {
+  $(element).parent().find('textarea').get(0).value = '';
+  let data = {"dataset_name": dataset_name, "campaign": campaign, "missing_nts": notes};
+  $.ajax({
+    type: "POST",
+    contentType: "application/json",
+    url: "/samples/missing_update",
+    data: JSON.stringify(data),
+  }).done(function (data) {
+    alert('Text reset in the database');
+  }).fail(function(data) {
+    alert('Not reset');
+  })
+}
+
+function saveNotesMissing(element, dataset_name, campaign) {
+  // Take the value of the text box
+  let notes = $(element).parent().find('textarea').get(0).value;
+  console.log(notes);
+  console.log($(element).parent().find('textarea').get(0));
+  let data = {"dataset_name": dataset_name, "campaign": campaign, "missing_nts": notes};
+  $.ajax({
+    type: "POST",
+    contentType: "application/json",
+    url: "/samples/missing_update",
+    data: JSON.stringify(data),
+  }).done(function (data) {
+    alert('Text saved in the database');
+  }).fail(function(data) {
+    alert('Error in saving');
```

Comment:
It would be better if you could also add `data` to the error message (might even be `data.data` or `data.message`, I don't remember where exactly message is in the data). If there is an error message, it's worth showing it to the user.